models/rule_helpers_test.go

```diff
@@ -221,8 +254,8 @@ func TestRules(t *testing.T) {
 	})
 }
 
-func createCreateRuleParams(templateName, channelID string) *models.CreateRuleParams {
-	return &models.CreateRuleParams{
+func createCreateRuleParams(templateName, channelID string, filter models.Filter) *models.CreateRuleParams {
```

Comment:
we can replace filter with filters here, in that case we can remove `emptyFilter` and if on 276 line.

Reply:
done